src/compiler/checker.ts

```diff
@@ -7521,6 +7569,12 @@ namespace ts {
                 return resolveErrorCall(node);
             }
 
+            // If the expression is of abstract type, then it cannot be instantiated.
+            var valueDecl = (expressionType.symbol ? expressionType.symbol.valueDeclaration : undefined);

```

Comment:
We usually use `expressionType.symbol && expressionType.symbol.valueDeclaration`